stock_partner_delivery_window/README.rst

```diff
@@ -44,6 +44,7 @@ Possible configurations are:
 
 * Any time: Do not postpone deliveries
 * Fixed time windows: Postpone deliveries to the next preferred time window
+* Weekdays: Postpone deliveries to the next weekday
```

Comment:
Do not edit directly `README.rst`, it is automatically generated from files in `./readme/` subfolder. `CONFIGURE.rst` is the one you are looking for.